statistics/table.go

```diff
@@ -324,8 +324,7 @@ func (coll *HistColl) GetRowCountByIntColumnRanges(sc *stmtctx.StatementContext,
 		}
 		return getPseudoRowCountByUnsignedIntRanges(intRanges, float64(coll.Count)), nil
 	}
-	result, err := c.GetColumnRowCount(sc, intRanges, coll.ModifyCount, true)
-	result *= c.GetIncreaseFactor(coll.Count)
+	result, err := c.GetColumnRowCount(sc, intRanges, coll.Count, true)
```

Comment:
We directly discard the increase factor?

Reply:
No. I put it in `GetColumnRowCount` to make it apply the increase factor first then add up the out-of-range part.
Because it's more reasonable to only apply the increase factor on the in-the-range part.